src/main/resources/project/ec_setup.pl

```diff
@@ -75,18 +75,18 @@
 
 # Data that drives the create step picker registration for this plugin.
 my %decommissionEnvironments = (
-	label     => "EF-Utilities - Decommission Environments",
-	procedure => "Decommission Environments",
+	label       => "EF-Utilities - Decommission Environments",
+	procedure   => "Decommission Environments",
 	description => "Decommission dynamic environments provisioned from pipeline, applications process or procedure",
-	category => "Deploy"
+	category    => "Deploy"
 );
 
-#my %createScheduleToTriggerGate = (
-#  label       => "EF-Utilities - createScheduleToTriggerGate",
-#  procedure   => "createScheduleToTriggerGate",
-#  description => "Create a schedule that trigger a gate at a particular time and date",
-#  category    => "Deploy"
-#);
+my %createScheduleToTriggerGate = (
+  label       => "EF-Utilities - createScheduleToTriggerGate",
```

Comment:
In perl world there is no any limitations to naming, but, we can use camelCase or underscore_naming if it's reasonable, for example, if project written in camelCase, we can use camelCase, if underscore, we can use underscore. Indentation and code formatting are much more important.